tests/test_journal.py

```diff
@@ -497,8 +514,28 @@ def test_submission(self, journal, openreview_client, test_client, helpers):
             weight=1
         ))
 
-        helpers.await_queue_edit(openreview_client, edit_id=paper_assignment_edge.id)
+        # immediately remove assignment of David Belanger
+        paper_assignment_edge.ddate = openreview.tools.datetime_millis(datetime.datetime.utcnow())
+        paper_assignment_edge = joelle_client.post_edge(paper_assignment_edge)
+
+        # wait for process function delay (5 seconds) and check no email is sent
+        messages = journal.client.get_messages(
+            to='david@mailone.com', subject='[TMLR] Assignment to review new TMLR submission Paper title UPDATED')
+        assert len(messages) == 0
+
+        # add David Belanger again
+        paper_assignment_edge = joelle_client.post_edge(openreview.Edge(invitation='TMLR/Reviewers/-/Assignment',
+            readers=[venue_id, f"{venue_id}/Paper1/Action_Editors", '~David_Belanger1'],
+            nonreaders=[f"{venue_id}/Paper1/Authors"],
+            writers=[venue_id, f"{venue_id}/Paper1/Action_Editors"],
+            signatures=[f"{venue_id}/Paper1/Action_Editors"],
+            head=note_id_1,
+            tail='~David_Belanger1',
+            weight=1
+        ))
 
+         # wait for process function delay (5 seconds) and check email has been sent
+        time.sleep(6)
```

Comment:
@melisabok in e0db1c32ff703053bd40401c452125e60ada11ac delay is change from 5 to 30
so for the test to pass, Ln 537 and 554 will need to change it to wait at least 30s
maybe it can be shorter than 30 seconds?

Reply:
Let me fix the test, sorry about this